double-conversion/utils.h

```diff
@@ -130,6 +130,9 @@ int main(int argc, char** argv) {
 #else
 #undef DOUBLE_CONVERSION_CORRECT_DOUBLE_OPERATIONS
 #endif  // _WIN32
+#elif defined(__ghs)
```

Comment:
This should go with the other defines to line 122.

Reply:
Done.